### ISSUE TITLE
Invalid schema validation on container apps template fails plan in 1.13

### ISSUE THREAD

**Issue by pgelinas** [Azure/terraform-provider-azapi#468]
There seems to be a new schema validation on 1.13 that is breaking our usage of azapi for container apps. We get the following error:

```
│ embedded schema validation failed: the argument "body" is invalid:
│ `properties.template.containers.0.resources.cpu` is invalid, expect
│ `integer` but got `float64`
```

However, having a float for the 'cpu' of this resource is totally fine per Azure spec. This is a major breaking change.

**Comment by ms-henglu** [comment]
Would you please share the configuration to help investigate?

**Comment by ms-henglu** [comment]
related issue: https://github.com/Azure/bicep-types-az/issues/1404

**Comment by ms-henglu** [comment]
Hi @pgelinas ,

The bug fix has been released in v1.13.1, would you please give it a try? Thanks!
